### DISCUSSION TITLE
Usuarios de la API

### DISCUSSION THREAD

**Discussion by ircsasw** [phpcfdi/api-descarga-masiva#17] [Q&A]
Se dijo que en un repositorio adicional se va a desarrollar el Front que consuma esta API, ¿se espera tener otros usuarios que hagan uso de la API desde sus propios desarrollos o es una API sólo para "nuestro" Front?

**Comment by celli33** [comment]
Me parece que se busca que se puedan desarrollar muchos fronts propios basados en esta API. Aunque no necesariamente tendría que estar en otro repo podríamos estructurar este mismo repo como: api-descarga-masiva/api/{laravel-app}, api-descarga-masiva/frontend/{front-app}. También me paree muy válido tener un repo separado. ¿Qué opinan?

**Comment by eclipxe13** [comment]
A mi parecer:

* Se busca tener una API que pueda ser consumida por otras apps o clientes, pero no como prioridad. La prioridad está en una API que pueda ser consumido por el front que hagamos en PhpCfdi.

* Se puede tener un repo separado o en el mismo. Recuerdo que Fernando prefería que fuera en un repo separado. También había escuchado esa recomendación por parte de César. No es una situación que no se pueda revertir, si de pronto se comienza por separado y luego se junta, o viceversa, no es gran problema.
